src/pages/AddToCartPage/AddToCartPage.jsx

```diff
@@ -1,12 +1,35 @@
 import { HorizontalCard } from "../../components/HorizontalCard/HorizontalCard";
 import { CartPrice } from "../../components/CartPrice/CartPrice";
+import { useCartContext } from "../../context/cart-context";
+import { Link } from "react-router-dom";
+import "./AddToCartPage.css";
 const AddToCartPage = () => {
+  const { cartState, cartDispatch } = useCartContext();
   return (
     <div className="mainbox">
       <div className="card-outerspace">
-        <HorizontalCard />
+        {cartState.cartList.map((items) => {
+          return <HorizontalCard item={items} />;
```

Comment:
You should also provide an ID whenever map is used

Reply:
yes lots of places I forgot to do that will fix all that in end of project will refactor all that in one go at the end